src/main/java/org/jabref/logic/l10n/Localization.java

```diff
@@ -48,13 +49,15 @@ private Localization() {
      * @param params Replacement strings for parameters %0, %1, etc.
      * @return The message with replaced parameters
      */
-    public static String lang(String key, String... params) {
+    public static <T> String lang(String key, T... params) {
```

Comment:
Isn't Object better here instead of `T`? Otherwise all `params` need to have the same type, don't they?

Reply:
Still possible to pass the second parameter as string like we did before

Reply:
But `lang("...", file, path)` is not possible since file and path are not of the same type T